File: reporters/test/artifacts/dotnet/mstest-vb-failing/CalculatorTests.vb
Imports Microsoft.VisualStudio.TestTools.UnitTesting

Namespace Calculator.Tests

    <TestClass>
    Public Class CalculatorTests

        <TestMethod>
        Public Sub Should_add_numbers_correctly()
            Assert.AreEqual(6, 2 + 3)
        End Sub

    End Class

End Namespace

File: reporters/test/artifacts/dotnet/nunit-passing/CalculatorTests.cs
using NUnit.Framework;

namespace Calculator.Tests
{
    [TestFixture]
    public class CalculatorTests
    {
        [Test]
        public void Should_add_numbers_correctly()
        {
            Assert.That(2 + 3, Is.EqualTo(5));
        }
    }
}
